### DISCUSSION TITLE
Where is config?

### DISCUSSION THREAD

**Discussion by janIpen** [iPortalTeam/ImmersivePortalsMod#543] [Q&A] [answered]
I am trying to make a non Euclidian geometry video, and the reduced portal rendering is breaking the immersion. It says disable it in 'config', but there is no such option in the pause menu, even when I click on 'options'. Please help me.

**Comment by qouteall** [comment, marked as answer]
Are you using Forge? The forge version does not have config gui. The config file is  `.minecraft/config/immersive-portals-client.toml`

**Reply by janIpen** [reply]
forge

**Reply by janIpen** [reply]
also my installation is at .../desktop/Minecraft Installations/Forge 1.16.4

**Reply by janIpen** [reply]
and the world is .../NEG Attempt II
Please give full directory using these

**Reply by froyo4u** [reply]
Should be `desktop/Minecraft Installations/Forge 1.16.4/config/immersive-portals-client.toml` and `desktop/Minecraft Installations/Forge 1.16.4/config/immersive-portals-server.toml`.

**Reply by janIpen** [reply]
thx

**Reply by janIpen** [reply]
And what should I use to open it?
Also I don't have desktop/Minecraft Installations/Forge 1.16.4/config/immersive-portals-server.toml

**Reply by janIpen** [reply]
and what do I change

**Reply by squendan998** [reply]
if you are on fabric then download modmenu then go on minecraft and click on the mods button and click on immersive portals then press the button on top of the issues button then click on client side and press reduced portal rendering and make sure it is set to no then press save and quit
